backend/oasst_backend/models/message_tree_state.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+from typing import Optional
+from uuid import UUID, uuid4
+
+import sqlalchemy as sa
+import sqlalchemy.dialects.postgresql as pg
+from sqlmodel import Field, Index, SQLModel
+
+# The types of States a message tree can have.
+
+
+class States(Enum):
+    INITIAL = "initial"
+    BREEDING_PHASE = "breeding_phase"
+    RANKING_PHASE = "ranking_phase"
+    READY_FOR_SCORING = "ready_for_scoring"
+    CHILDREN_SCORED = "children_scored"
+    FINAL = "final"
+
+
+VALID_STATES = (
+    States.INITIAL,
+    States.BREEDING_PHASE,
+    States.RANKING_PHASE,
+    States.READY_FOR_SCORING,
+    States.CHILDREN_SCORED,
+    States.FINAL,
+)
```

Comment:
could this be replaced by either `tuple(States)` or `tuple(s.value for s in States)` ?

Reply:
Sure